draft-kazuho-httpbis-priority.md

```diff
@@ -300,6 +302,84 @@ receipt, a client that supports header-based prioritization MUST close the
 connection with a protocol error. Non-supporting clients will ignore this
 extension element (see {{!RFC7540}}, Section 5.5).
 
+# Reprioritization
+
+Once a client sends a request with a priority, circumstances might change and
```

Comment:
> Once a client sends a request with a priority

I think we might want to strike "with a priority," as even a request without the Priority header field carries an implicit priority.

Reply:
excellent point! fixed